### ISSUE TITLE
Once we search any item in Peoplepicker  ,option items are displayed and option items are faded on mouse hover in IE 11 high contrast mode

### ISSUE THREAD

**Issue by v-ajrai** [microsoft/fluentui#12896]
Once we search any item in Peoplepicker  ,option items are displayed and option items are faded on mouse hover in IE 11 high contrast mode
### Environment Information

OS: Win 10
Browser: IE11

### Describe the issue:
 Peoplepicker option items should not fade on mouse hover in IE 11 High contrast mode.Once peoplepicker option items expanded  through mouse ,the options are faded out on mouse hover.

### Please provide a reproduction of the issue in a codepen:
https://developer.microsoft.com/en-us/fluentui#/controls/web/peoplepicker

![Bug_id_1803959_external](https://user-images.githubusercontent.com/62233982/80474959-c5b69180-8965-11ea-9249-32701e2079ae.png)
![Uploading Bug_id_1803959_external_highcontrast_black.png…]()

#### Actual behavior:

Once we search any item in Peoplepicker  ,option items are displayed and option items are faded on mouse hover in IE 11 high contrast mode

#### Expected behavior:

Once we search any item in Peoplepicker  ,option items are displayed and option items should not be faded on mouse hover in IE 11 high contrast mode

**Comment by aneeshack4** [comment]
@v-ajrai This doesn't repro on the new Edge Chromium. You can provide your own styling specific with high contrast selectors to work around this. Since this isn't a breaking issue for IE 11 and it's a minor scenario, we're not going to address this right now.
